xdebug_com.c

```diff
@@ -25,165 +25,205 @@
 # include <sys/select.h>
 #endif
 #ifndef PHP_WIN32
-#include <unistd.h>
-#endif
-
-#ifdef PHP_WIN32
-# include <process.h>
-# include <direct.h>
-# include "win32/time.h"
-#define PATH_MAX MAX_PATH
-#else
+# include <unistd.h>
 # include <sys/socket.h>
-# include <netinet/in.h>
 # include <netinet/tcp.h>
-# include <arpa/inet.h>
 # include <netdb.h>
+#else
+# include <process.h>
+# include <direct.h>
+# include "win32/time.h"
+# undef UNICODE
```

Comment:
The pragma is, according to the documentation, a directive to the linker that Ws2_32.lib should be included. This can also be configured in the IDE, but having it here also should not hurt.